app/models/bet.rb

```diff
@@ -27,4 +23,28 @@ def amount_owed(lost_bet, won_bet)
   def bet_total
     user_bets.pluck(:amount_bet).sum
   end
+
+  private
+
+  def resolve_debt(lost_bet, won_bet)
+    existing_debt = Debt.find_by(debtor: lost_bet.user, creditor: won_bet.user)
+    if existing_debt
+      update_debt(existing_debt, lost_bet, won_bet)
```

Comment:
I kind of feel like this is out of scope for this PR ...... I really dont want to worry about duplcated creditor -> debtor debts yet